doc/lightning-fundchannel.7.txt

```diff
@@ -0,0 +1,54 @@
+LIGHTNING-FUNDCHANNEL(7)
+========================
+:doctype: manpage
+
+NAME
+----
+lightning-fundchannel - Command for establishing a lightning channel.
+
+SYNOPSIS
+--------
+*fundchannel* 'id' 'satoshi'
+
+DESCRIPTION
+-----------
+The *fundchannel* RPC command opens a payment channel with a peer by commiting
+a funding transaction to the blockchain as defined in BOLT #2.  Once the
+transaction is confirmed, normal channel operations may begin.  Readiness is
+indicated by *listfunds* reporting a 'short_channel_id' for the channel.
+Channel status can then be monitored using *listchannels*.
+
+'id' is the peer id obtained from *connect*.
+
+'satoshi' is the amount in satoshis taken from the internal wallet to fund the
+channel.  The string 'all' can be used to specify all available funds.  This value
+must be greater than the dust limit, currently set to 546.  And it must be less
+than 1<<24 (approximately 0.16778 BTC).
+
+RETURN VALUE
+------------
+On success, the 'tx' and 'txid' of the transaction is returned, as well as the
+'channel_id' of the newly created channel.  On failure, an error is reported
+and the channel is not funded.
+
+The following error codes may occur:
+
+* -1. Catchall nonspecific arror.
+
+The above error may include a descriptive message indicating:
+
+* The 'id' is invalid.
```

Comment:
Yes, we should probably start an effort at some point to give unique error codes for some of the more complex commands. Since it is a JSON-RPC change this will probably have to be done pre-0.6 release too.  This is fine for now though; we can update the docs later when we finally have unique error codes.